README.md

```diff
@@ -1,4 +0,0 @@
-# idp-mgmt-docs
```

Comment:
Did you mean to delete this file? It would be nice if we could have the README.md file explain the two instances of the repo and have the links to the two sets of docs.

Reply:
I renamed it to README.adoc as ebook needed it.
I will try to add a README.md but asciidoc made strange things with it.

Reply:
@robinbobbitt README.md added, README.adoc renamed as index.adoc